spec/draper/configuration_spec.rb

```diff
@@ -6,20 +6,40 @@ module Draper
       Draper.configure { |config| expect(config).to be Draper }
     end
 
-    it 'defaults default_controller to ApplicationController' do
-      expect(Draper.default_controller).to be ApplicationController
-    end
+    describe '#default_controller' do
+      it 'defaults default_controller to ApplicationController' do
+        expect(Draper.default_controller).to be ApplicationController
+      end
+
+      it 'allows customizing default_controller through configure' do
```

Comment:
Did you want to write `allows customizing default_controller through configuration`?

Reply:
Or even, `through Draper#configure`, `through configure method`

Reply:
I'm not sure :thinking:. I've just moved this old tests into a context, maybe these changes are out the scope of this pr?